crates/bifrost/src/loglet.rs

```diff
@@ -150,6 +153,17 @@ pub trait Loglet: Send + Sync {
     async fn seal(&self) -> Result<(), OperationError>;
 }
 
+#[derive(Default, Clone, Copy, PartialEq, Eq)]
+pub enum FindTailAttr {
+    /// Finds the durable tail of the loglet.
+    #[default]
+    Durable,
+    /// Approximate find tail of the loglet. This should cheap to run based on the
+    /// loglet implementation. If an efficient approximation is not possible, it must
+    /// fall back to a durable check.
```

Comment:
I might have mislead you by suggesting approx as a good name. Instead, I think explaining this in terms of the fact we might be returning a stale value of the tail is better. The enum variant can be `StaleAllowed` or `Fast`